contrib/lang/ruby/packages.el

```diff
@@ -41,15 +47,82 @@ which require an initialization must be listed explicitly in the list.")
 (defun ruby/init-flycheck ()
   (add-hook 'enh-ruby-mode-hook 'flycheck-mode))
 
+(defun ruby/init-projectile-rails ()
+  (use-package projectile-rails
+    :defer t
+    :init (progn
+            (add-hook 'projectile-mode-hook 'projectile-rails-on))
+    :config (progn
+              (spacemacs|diminish projectile-rails-mode " ⇋" " R")
+              ;; GOTO commands
+              (evil-leader/set-key "mrf" 'projectile-rails-goto-file-at-point)
```

Comment:
most of the goto and find commands are related to concept that are specific to rails project, what about dropping the `r` prefix and have sequences like `mgm` (go to model), `mgc` (go to controller) etc.? that would save one keystroke and not pollute the main ruby mode...